observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -410,7 +410,10 @@ object TargetObsList {
 
       def renderObsBadge(obs: ObsAttached): TagMod =
         ObsBadge(
-          ObsSummary(obs.id, obs.name.orEmpty),
+          ObsSummary(id = obs.id,
+                     name = obs.name,
+                     observationTarget = None
+          ), // FIXME Add the target id
```

Comment:
Regarding the comment: I don't think we want the target id here since the badge is already nested within the target (or asterism). (According to the spec, we have to render the observation badge differently in different views)

Reply:
👍